src/test/regress/expected/multi_mx_create_table.out

```diff
@@ -53,7 +53,16 @@ CREATE OPERATOR citus_mx_test_schema.=== (
     HASHES, MERGES
 );
 SET search_path TO public;
+SHOW server_version \gset
```

Comment:
do we need this if / else?

Can't we always do what PG 16 does in the test? I don't think anyone really cares what is the `locale` that is  used. The criticalness of this test is more around we as _a_ locale.

If that is the case, can you change the test in the main branch with a separate PR?

Reply:
Apparently we need it because daticuprovider column is not available pre PG16